docs/operations/hotplug_interfaces.md

```diff
@@ -100,6 +100,34 @@ You can now check the VMI status for the presence of this new interface:
 kubectl get vmi vmi-fedora -ojsonpath="{ @.status.interfaces }"
 ```
 
+### Migration based hotplug
+In case your cluster doesn't meet the requirements, it's possible to hotplug an interface by migrating the VM.
```

Comment:
```suggestion
In case your cluster doesn't use the Multus version as specified above, it's possible to hotplug an interface by migrating the VM.
```

Reply:
It's not only the multus version: if you have the correct multus version, but didn't deploy the dynamic networks controller, it still won't work.

Thus, I think the original suggestion was preferable - while a bit vague, it is specific enough to be correct.